src/styles/levelSelect.scss

```diff
@@ -63,5 +65,30 @@
   display: inherit;
   grid-column: 3;
   justify-self: left;
-  padding-bottom: 2.5px;
+  margin-bottom: 13px;
+  margin-left: 5px;
+}
+
+.left-arrow {
```

Comment:
Instead of setting the arrow positions with absolute positioning and pixel values can we use grid/flexbox? For example, maybe a grid would contain three things: left arrow, dropdown, and right arrow. It looks like your `grid-container` class already does this. The CSS might look something like this: `justify-content: space-between`